fern/definition/medical/document.yml

```diff
@@ -265,6 +365,63 @@ types:
           code: "62479008"
           display: Diagnoses
 
+
+  UploadDocumentReference:
```

Comment:
Internally we require that uploaded document references have a description, type, and context, not to the exclusion of other resource types. We should be able to use Fern import types here and then ssimplify this greatly too. 
We also need to be able to capture the logic that we require certain properties but that other remaining ones are optional and permitted.